### DISCUSSION TITLE
when usb powerd , it not work. ?

### DISCUSSION THREAD

**Discussion by peng2e** [gopro/labs#845] [Q&A] [answered]
my tool is max, i want  recoding in my car when usb powerd。

As shown below when usb powerd ,it not work ，need press the power button can recoding. 
but no usb powerd it work.
![image](https://github.com/gopro/labs/assets/4724507/ad27ba75-2796-42e4-8b37-30021bf82efb)

**Comment by dnewman-gpsw** [comment, marked as answer]
I gather you want to boot the camera also when power is applied?  For that you need to add 
![image](https://github.com/gopro/labs/assets/10856084/c9392add-0914-467a-a226-b34c76424da3)
